README.md

```diff
@@ -1,47 +1,106 @@
+
+We use it `css-loader` to generate a CSS token on a HTML element. For example, the builder generate a CSS token based on app vendor, name and major version. Like `container` token declared in category-menu, generate the classname `vtex.category-menu-2-x-container`.
+
+Below, we describe the tokens, their explanation and the component where it is located.
+
+| Token name         | Component          | Description                                            |
+| ------------------ | ----------         |------------------------------------------------------- |
+| `container`        | [index](https://github.com/vtex-apps/category-menu/blob/master/react/index.js)           | The main container of Category Menu for all sizes                        |
+| `mobile`          | [index](https://github.com/vtex-apps/category-menu/blob/master/react/index.js)            | The main container of Category Menu for mobile size                                        |
+| `animation`            | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)            | Category menu mobile sidebar animation         |
+| `sidebarOpen`              | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)            | Active when the sidebar is opened                        | 
+| `sidebarScrim`          | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)  | the sidebar shadow backdrop content                           |
+| `sidebar`     | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)   | The main container of sidebar                     |
+| `sidebarHeader`     | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)  | The sidebar header container                     |
+| `sidebarContent`  | [SideBar](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBar.js)   | The sidebar content container              | 
+| `firstLevelLink`    | [ItemContainer](https://github.com/vtex-apps/category-menu/blob/master/react/components/ItemContainer.js)          | First level of the Category Menu, the main categories links                      |
+| `secondLevelLink `          | [ItemContainer](https://github.com/vtex-apps/category-menu/blob/master/react/components/ItemContainer.js)          | Second level of the Category Menu, the subcategories links                        |
+| `itemContainer`    | [CategoryItem](https://github.com/vtex-apps/category-menu/blob/master/react/components/CategoryItem.js)            | Main container of a category item/department                                    |
+| `sidebarItem`     | [SideBarItem](https://github.com/vtex-apps/category-menu/blob/master/react/components/SideBarItem.js)   | The container of a sidebar category item/department     |
+
+To override the default CSS, you need to import `styles` on your manifest:
```

Comment:
You also need to add on dependencies. I think this text needs to be before the table